cheroot/workers/threadpool.py

```diff
@@ -3,7 +3,6 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
```

Comment:
Please don't add completely unrelated changes. This makes it harder to do reviews.

Reply:
Seems like this was either done by precommit hook or flake8 autoformatter used for satisfying precommit hook. Will check again.

Reply:
removed line